wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+---
+tags:
+  - UK Community Cup 7
+  - UKCC 7
+  - UKCC 7
+---
+
+# UK Community Cup 7
+
+![UKCC 7 logo](img/banner.png)
+
+The seventh edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.
```

Comment:
generally all wiki articles introduce the main topic as the subject of the first sentence, and bold it too. something like this would be better
```suggestion
The **UK Community Cup 7** (***UKCC 7***) is the seventh edition of the bi-annual UK Community Cup, a tournament series for those who live in or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th of October, 2019.
```
maybe add flag icons before the countries too, idk

Reply:
also I didn't realise this tournament was already over. so it should be written in past tense (this `was` the seventh edition of ...)